packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -113,26 +121,51 @@ export class Analyzer {
       // The jsPath appears to come out of the ts API with unix
       // separators; since sourcePath uses OS separators, normalize
       // this so that all our model paths are OS-native
-      jsPath: path.normalize(jsPath) as PackagePath,
+      jsPath: absoluteToPackage(
+        path.normalize(jsPath) as AbsolutePath,
+        this.packageRoot
+      ),
       sourceFile,
     });
 
+    this.programContext.currentModule = module;
+
     for (const statement of sourceFile.statements) {
       if (ts.isClassDeclaration(statement)) {
-        if (isLitElement(statement, this.checker)) {
+        if (isLitElement(statement, this.programContext)) {
           module.declarations.push(
-            getLitElementDeclaration(statement, this.checker)
+            getLitElementDeclaration(statement, this.programContext)
           );
-        } else {
+        } else if (isExported(statement)) {
           module.declarations.push(
             new ClassDeclaration({
               name: statement.name?.text,
               node: statement,
             })
           );
         }
+      } else if (ts.isVariableStatement(statement) && isExported(statement)) {
```

Comment:
What is this adding exactly and why?

Reply:
> In order to test this, also adds analysis of VariableDeclaration and a suite of type tests against a module with variously typed variable declarations.

`VariableDeclaration` is one of a few more we ultimately need to add to the analyzer to fully generate the CEM: https://github.com/webcomponents/custom-elements-manifest/blob/main/schema.d.ts#L162